doc/howtos/rdtraining/04_basicmodel.rst

```diff
@@ -0,0 +1,298 @@
+.. _howto/rdtraining/04_basicmodel:
+
+===============================
+Part 4: Models And Basic Fields
+===============================
+
+At the end of the :ref:`previous chapter <howto/rdtraining/03_newapp>`, we were able to create an Odoo
+module. However, at this point it is still an empty shell which doesn't allow us to store any data.
+In our real estate module, we want to store the information related to the properties
+(name, description, price, living area...) in a database. The Odoo framework provides tools to
+facilitate the interaction with the database.
+
+Before moving forward on the excercise, make sure the ``estate`` module is installed, i.e. it
+must appear as 'Installed' in the Apps list.
+
+Object-Relational Mapping
+=========================
+
+**Reference**: the documentation related to this topic can be found in the
+:ref:`reference/orm/model` API.
+
+.. note::
+
+    **Goal**: at the end of this section, the table ``estate_property`` is created:
+
+    .. code-block:: text
+
+        $ psql -d rd-demo
+        rd-demo=# SELECT COUNT(*) FROM estate_property;
+        count
+        -------
+            0
+        (1 row)
```

Comment:
It should be seen this way: the goal gives you what should have at the end of the section. I agree it is low-level and we didn't explain how to play with psql, but it seems the most universal and simple way to make sure the table is created. We could also use `\d`, but IIRC the technical fields are already created, so that would be more information to give.